typescript error with @grpc/proto-loader: 0.6.11

Hi, I'm getting typescript compilation with new version of @grpc/proto-loader: 0.6.11

I have a depencency on 'googleapis'. Previous version resolved was @grpc/proto-loader: 0.6.9 not causing issue. Now it's resolving to 0.6.11 with the following typescript compilation error:

node_modules/@google-cloud/pubsub/build/protos/protos.d.ts:15:23 - error TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.

15 import * as Long from "long";
~~~~~~

node_modules/google-gax/build/protos/iam_service.d.ts:17:23 - error TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.

17 import * as Long from 'long';
~~~~~~

node_modules/google-gax/build/protos/operations.d.ts:17:23 - error TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.

17 import * as Long from 'long';

#### Environment details
OS: centos8
Node.js version: 14.16.0
npm version: 6.14.11
googleapis version: 100.0.0 (same issue in v95)

#### Steps to reproduce
npm i
npx tsc

Strangely the resolved version on my ubuntu machine is still 0.6.9, but on my centos machine, it resolves to 0.6.11. In the attached repro project, I have explicitely added "@grpc/proto-loader": "0.6.11" to trigger the issue.

Note that I've entered the same issue in @googleapis because I'm not sure where the issue come from.

Thanks!

[repro.tar.gz](https://github.com/grpc/grpc-node/files/8616759/repro.tar.gz)

--- a/packages/proto-loader/package.json
+++ b/packages/proto-loader/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@grpc/proto-loader",
-  "version": "0.6.11",
+  "version": "0.6.12",
   "author": "Google Inc.",
   "contributors": [
     {
@@ -47,7 +47,7 @@
   "dependencies": {
     "@types/long": "^4.0.1",
     "lodash.camelcase": "^4.3.0",
-    "long": "^4.0.0 || ^5.2.0",
+    "long": "^4.0.0",
     "protobufjs": "^6.10.0",
     "yargs": "^16.2.0"
   },

